target/linux/ramips/dts/mt7621_dlink_dir-878-r1.dts

```diff
@@ -0,0 +1,8 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "mt7621_dlink_dir-8xx-r1.dtsi"
```

Comment:
Does this compile? It seems we don't have a file of this name?

Reply:
Before compiling locally I ran the command:
"git checkout v21.02.1"
taken from the instructions of building openwrt:
https://openwrt.org/docs/guide-developer/toolchain/use-buildsystem

It seems like branch 21.02.1 has these files. 
interesting why these files are not here on the mirror branch.
any thoughts?

Reply:
@lior-pugatch Your PR is against master, right? You need to base it on master.

Master has [mt7621_dlink_dir-8xx.dtsi ](https://github.com/openwrt/openwrt/blob/master/target/linux/ramips/dts/mt7621_dlink_dir-8xx.dtsi) (note: no '-r1' suffix), no `mt7621_dlink_dir-8xx-r1.dtsi` though as mans0n says.

Reply:
When is it expected to have branch 21.02.1 merged to master?
is branch 21.02.1 considered "newer" than master?

Reply:
What is your interest in release 21.02?
Do you want see your DIR-878 R1 device supported by a 21.02 release?

Then do anything what the reviewers ask, and wait it for landing in `master`.
After that, pick that commit on `master` and _backport_ to the `openwrt-21.02` branch! With `git cherry-pick -x` and resolve the conflict (like this `.dtdi` mismatch).

For example see another backporting PR #4630:
* support for Xiaomi AIoT Router AC2350 landed first to `master` in commit 6c148116f778bfd2db88476cee4753b32538eafe from PR #4161, 
* and then was backported which was landed to `openwrt-21.02` branch as commit 06547e0a58daf768ff9776339cd31231dcd5c0ea.

Reply:
@lior-pugatch Device support PRs are only accepted against master. As @xabolcs explained you can then request a backport to a stable release like 21.02. But only then; it needs to be in master first since that's where development happens.

Reply:
Your work and test should be based on top of the master branch. Please do not just copy the patch based on the stable branch and say it's done.